pom.xml

```diff
@@ -562,6 +562,29 @@ under the License.
                         <target>1.7</target>
                     </configuration>
                 </plugin>
+
+                <!-- Fail a build if it uses any Java features introduced after 1.7. -->
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>animal-sniffer-maven-plugin</artifactId>
+                    <version>1.15</version>
+                    <configuration>
+                        <signature>
+                            <groupId>org.codehaus.mojo.signature</groupId>
+                            <artifactId>java17</artifactId>
```

Comment:
I think a later pull request upgrades rya back to java 8 (an upgrade that was lost somehow when we ported to apache).  Can you update this accordingly?  Then I will merge.